### PR TITLE
update and extend metrics documentation

### PR DESCRIPTION
**Pull request by brancz**
We have done some renaming and additions to the metrics, so updating the docs and extending it with metrics type.

@fabxc @ghodss

### CODE REVIEW

**Comment by brancz** [comment]
I signed it!

**Comment by fabxc** [comment]
You've signed this at least 3 times now. Something seems to be not working here correctly.

File: README.md [hunk 1]
```diff
@@ -24,12 +24,13 @@ additional metrics!
...
-| deployment_replicas | name=&lt;deployment-name&gt; namespace=&lt;deployment-namespace&gt; |
-| deployment_replicas_available | name=&lt;deployment-name&gt; namespace=&lt;deployment-namespace&gt; |
-| container_restarts | name=&lt;container-name&gt; namespace=&lt;pod-namespace&gt; pod_name=&lt;pod-name&gt; |
+| Metric name| Metric type | Labels/tags |
+| ---------- | ----------- | ----------- |
+| node_info | Gauge | node=&lt;node-address&gt; kernel_version=&lt;kernel-version&gt; os_image=&lt;os-image-name&gt; container_runtime_version=&lt;container-runtime-and-version-combination&gt; kubelet_version=&lt;kubelet-version&gt; kubeproxy_version=&lt;kubeproxy-version&gt; |
+| node_status_ready| Gauge | node=&lt;node-address&gt; condition=&lt;true\|false\|unknown&gt; |
+| deployment_replicas | Gauge | name=&lt;deployment-name&gt; namespace=&lt;deployment-namespace&gt; |
+| deployment_replicas_available | Gauge | name=&lt;deployment-name&gt; namespace=&lt;deployment-namespace&gt; |
+| pod_container_restarts | Counter | container=&lt;container-name&gt; namespace=&lt;pod-namespace&gt; pod=&lt;pod-name&gt; |
```
Comment (fabxc) [inline README.md]: Any particular reason for switching to escaped `<` and `>` or is your editor doing that?
This is fine for now. As we grow the metrics, we should probably break this out by collector.

One line per label will make it more readable for a lot of cases.

Comment (brancz) [inline README.md]: makes sense, yes I'll do one line per label and replace to non-escaped, just stayed with it because it was used before

Comment (brancz) [inline README.md]: nvm escaped is required because it is otherwise interpreted as an html tag

Comment (brancz) [inline README.md]: Yep, much more readable with line breaks. And breaking it out per collector should be done once we do add more metrics IMO.

**Comment by fabxc** [comment]
`name` -> `deployment` in deployment metric labels.

**Comment by brancz** [comment]
@fabxc fixed

**Comment by fabxc** [comment]
👍

**Pull request merged**